luigi_tools/task.py

```diff
@@ -52,6 +53,17 @@ def __init__(self, *args, **kwargs):
         if recursive_check(self):
             apply_over_outputs(self, target_remove)
 
+        if isinstance(self, luigi.WrapperTask) and self.rerun:
+
+            def recursive_target_remove(target, *args, **kwargs):
```

Comment:
Is this needed? WrapperTask has no output, and its complete() depends only on its requires().